integrations/observability/grafana/resources.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: observability
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana-lgtm
+  namespace: observability
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana-lgtm
+  template:
+    metadata:
+      labels:
+        app: grafana-lgtm
+    spec:
+      containers:
+        - name: grafana-lgtm
+          image: grafana/otel-lgtm:latest
+          ports:
+            - containerPort: 3000
+            - containerPort: 4317
+            - containerPort: 4318
+            - containerPort: 9411
+          volumeMounts:
+            - name: otelcol-config
+              mountPath: /otel-lgtm/otelcol-config.yaml
+              subPath: otel-collector-grafana.yaml
```

Comment:
Why are we using `subpath` in here? and should the `mountPath` be a file instead of a directory?

Reply:
There are more files in the `/otel-lgtm/` directory, I think it overrides the whole directory without `subPath` and `mountPath` being a file. Let me confirm it.

Reply:
We need it as it is, it overrides the whole directoy.